lntest/itest/lnd_test.go

```diff
@@ -13155,7 +13155,8 @@ func testAbandonChannel(net *lntest.NetworkHarness, t *harnessTest) {
 
 	ctxt, _ := context.WithTimeout(ctxb, channelOpenTimeout)
 	chanPoint := openChannelAndAssert(
-		ctxt, t, net, net.Alice, net.Bob, channelParam)
+		ctxt, t, net, net.Alice, net.Bob, channelParam,
```

Comment:
nit: maybe you wanted to add a param but deleted it? (sorry, this may be a stylistic thing I'm not familar with)

Reply:
Just updating older code to conform to the new style guidelines.